packaging/ubuntu-trusty/debian/changelog

```diff
@@ -1,3 +1,11 @@
+amazon-ecs-init (1.18.0-1) trusty; urgency=medium
+
+    * Cache Agent version 1.18.0
+    * Bundle ECS Agent tarball in package
+    * Download agent based on the partition
+    * Mount Docker plugin files dir
+   -- Haikuo Liu <haikuo@amazon.com>  Fri, 04 May 2018 13:30:22 -0800
```

Comment:
nit: Can you add an extra empty line between the entries and the maintainer name?